scripts/configs/PKGBUILD.template

```diff
@@ -5,34 +5,24 @@
 # this could lead to a loading issue if a user is loading the page exactly during the
 # upgrade, but so far there is nothing we can do about it (we would need to coordinate multiple nginx).
 
-targets=(
-  "centos"
-  "ubuntu"
-)
 pkgname="<%- htmlWebpackPlugin.options.name %>"
 pkgver="<%- htmlWebpackPlugin.options.version %>"
 pkgrel="<%- htmlWebpackPlugin.options.pkgRel %>"
 pkgdesc="<%- htmlWebpackPlugin.options.description %>"
-pkgdesclong=(
-  "<%- htmlWebpackPlugin.options.description %>"
-)
 maintainer="Zextras (packages@zextras.com)"
-arch="all"
-license=("spdx:AGPL-3.0-only")
+arch=("x86_64")
+license=("AGPL-3.0-only")
+copyright=(
+  "2022, Zextras <https://www.zextras.com>"
+)
```

Comment:
Something is wrong with this line, it throws a ParseError during the build of webpack. It might be related to the "<" char and HtmlWebpackPlugin used to replace the dynamic values @zextras/smoky-beans